sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncyptedBlockBlobAPITest.groovy

```diff
@@ -1489,6 +1490,7 @@ class EncyptedBlockBlobAPITest extends APISpec {
         }
     }
 
+    @IgnoreIf( { getEnv().serviceVersion != null } )
```

Comment:
Does the `getEnv().serviceVersion` not get overridden by the testing policy?

Reply:
It's either null or something depending on env.
https://github.com/kasobol-msft/azure-sdk-for-java/blob/2eb4f25a831128d403913179cd8ff33b4c8b44c9/sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java#L68-L77